print/cups-ppdc/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       legacysupport 1.1
+PortGroup       github 1.0
+
+github.setup    apple cups 2.3.3 v
+name            cups-ppdc
+categories      print textproc
+license         Apache-2
+maintainers     nomaintainer
+description     Compiler for PostScript Printer Description (PPD) files
+long_description \
+    cups-ppdc is a collection of utilities taken from the Common UNIX Printing System (CUPS) \
+    for working with PostScript Printer Description (PPD) files\; \
+    PPD files can be used to set up a printer when proprietary drivers are not available. \
+    See the ppdcfile(5) man page for information on the PPD file format.
+
+homepage        https://github.com/apple/cups
+
+checksums       rmd160  676fec5542a7ee80111011757589fe44f7354c29 \
+                sha256  fe381d941907011e0f2323dce93f0dc4a60cca9fa56b5d896288819cb933f074 \
+                size    8148606
+
+patchfiles      patch-static-targets.diff \
+                patch-configure.diff \
+                patch-cups-hash-sha224.diff \
+                patch-cups-usersys-ssl.diff \
+                patch-cups-getifaddrs.diff \
+                patch-cups-tempfile.diff
+
+platform darwin 8 {
+    patchfiles-append \
+                patch-cups-transcode-iconv.diff
+}
+
+post-patch {
+    # Fix broken build with old GCC and unrecognized flag warnings with new GCC
+    reinplace "s|@PIEFLAGS@||" Makedefs.in
+
+    # <os/availability.h> introduced in 10.12
+    if {${os.platform} eq "darwin" && ${os.major} < 16} {
+        reinplace "s|<os/availability.h>|<AvailabilityMacros.h>|" cups/versioning.h
+    }
+}
+
+configure.args-append   --disable-dnssd
+configure.args-append   --disable-gssapi
+configure.args-append   --disable-ssl
+configure.args-append   --disable-shared
+configure.args-append   --with-bundledir=""
+configure.cflags-append -fno-stack-protector
+configure.cxxflags-append -fno-stack-protector
+if {${os.platform} eq "darwin" && ${os.major} < 10} {
+    configure.cflags-append -U__BLOCKS__
+    configure.cxxflags-append -U__BLOCKS__
+}
+
+post-configure {
+    platform darwin 8 {
+        # res_init() symbol is found, but the prototype is never included due
+        # to broken resolv.h on Tiger. Don't need networking so just tell the
+        # build we've never heard of res_init().
+        reinplace "s|#define HAVE_RES_INIT 1|/* #undef HAVE_RES_INIT */|" config.h
+    }
+}
+
+build {
+    exec -ignorestderr make -C ${worksrcpath}/cups
```

Comment:
I am not 100% sure, but I don't think we typically use `exec` for this kind of things... most Portfiles use  `system -W ${build.dir} "make -C ....."`, does that not work here?

Reply:
I was just figuring out how to do multiple make commands, will switch to `system`.